### ISSUE TITLE
Confusion of the percentage definition

### ISSUE THREAD

**Issue by bcjohnblue** [cypress-visual-regression/cypress-visual-regression#270]
The result of `percentage` makes me confused.

It says `0.2` means the pixel difference is `0.2%` from [README.md](https://github.com/cypress-visual-regression/cypress-visual-regression/blob/master/README.md#-examples).
```
cy.get('h1').compareSnapshot('homePage', 0.2) // will compare only the image of h1 element and fail only if the percentage of pixels that are different is bigger than 0.2%
```

However, the `percentage` doesn't multiple by `100` in [source code](https://github.com/cypress-visual-regression/cypress-visual-regression/blob/70afd46077025d2ef32d742dca4bffd4158fb93e/src/plugin.ts#L152). I think the range of `percentage` is from 0 to 1 now.

```js
const percentage = (mismatchedPixels / diffImage.width / diffImage.height) ** 0.5
```

In conclusion, `0.2` should not translated to `0.2%`, it means`20%`.

------

By the way, I suggest to remove `** 0.5` when calculate `percentage`. It seems  https://github.com/haim-io/cypress-image-diff/issues/184 has similar request too.
